js/index.js

```diff
@@ -1 +1,110 @@
 // Your code goes here
+
+/****************
+ * 
+ * Main Nav
+ * 
+ ***************/
+function hideNav() {
+  const navBar = document.querySelector(".main-navigation");
+  // console.log(navBar);
+
+  window.addEventListener("scroll", function (event) {
+    let currentYoff = window.scrollY;
+    let navClasses = navBar.classList;
+
+    if (currentYoff > 90 && !navClasses.contains("scrolled")) {
+      navClasses.add("scrolled");
+    } else if (currentYoff < 90) {
+      navClasses.remove("scrolled");
+    }
+  });
+};
+hideNav();
+
+const navItems = document.querySelectorAll('.nav-link');
+
+navItems.forEach(navItem => navItem.addEventListener('click', (event) => {
+  event.preventDefault();
+}));
+
+navItems.forEach(navItem => navItem.addEventListener('mouseover', (event) => {
+  navItem.style.color = "lightgrey";
+}));
+navItems.forEach(navItem => navItem.addEventListener('mouseout', (event) => {
+  navItem.style.color = "";
+}));
+
+/****************
+ * 
+ * content section
+ * 
+ ***************/
+const contentSection = document.querySelector(".content-section");
+contentSection.addEventListener("dblclick", (e) => {
+  console.log("Double clicked content");
+});
+
+/****************
+ * 
+ * inverse content 
+ * 
+ ***************/
+const inverseContent = document.querySelectorAll(".inverse-content div");
+// console.log(inverseContent);
+inverseContent.forEach(item => () => {
```

Comment:
Nice job making everything draggable and accessing all of them withe the forEach! 👍